modules/compute/container-registry/README.md

```diff
@@ -0,0 +1,91 @@
+# Container Registry
+
+This module deploys Container Registry (Microsoft.ContainerRegistry/registries) and optionally available integrations.
+
+## Description
+
+{{ Add detailed description for the module. }}
```

Comment:
You'll need to add a more detailed description here too 😏

Reply:
Thank you for spotting that!